src/main/java/secondmind/Storage.java

```diff
@@ -0,0 +1,122 @@
+package secondmind;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Storage {
+    private final String DATA_FILE_PATH;
+
+    public Storage(String DATA_FILE_PATH) {
+        this.DATA_FILE_PATH = DATA_FILE_PATH;
+    }
+
+    public void appendToFile(String data, int taskCount) throws IOException {
+        File f = new File(DATA_FILE_PATH);
+        FileWriter fw = new FileWriter(f, true);
+        fw.write(data + "\n");
+        fw.close();
+    }
+
+    public void updateTaskInDataFile(int taskNumber, boolean done, int taskCount) throws FileNotFoundException, IOException {
+        int lineNumber = 1;
+        File oldFile = new File(DATA_FILE_PATH);
+        File newFile = new File("./tempDataFile.txt");
+        FileWriter fw = new FileWriter("./tempDataFile.txt", true);
+        Scanner s = new Scanner(oldFile);
+        while (s.hasNext()) {
+            String currentLine = s.nextLine();
+            if (lineNumber == taskNumber) {
+                StringBuilder sb = new StringBuilder();
+                //Add task type and "|" to stringbuilder
```

Comment:
1. Definitely a great thing adding comments to explain lines here and there!
2. Coding conventions suggest a space after `//` and `*` if doing code blocks